src/context/context.interface.ts

```diff
@@ -0,0 +1,24 @@
+import { AuthFormData } from "src/common/interface";
+import { User } from "src/modules/User";
+
+export interface GlobalState {
+  user?: User;
+  loading: boolean;
+}
+
+export enum Types {
```

Comment:
Error state?

Reply:
Not needed we will be showing error toast in that case so as to avoid storing and managing error state in our application.